astropy/coordinates/baseframe.py

```diff
@@ -588,15 +723,15 @@ def represent_as(self, new_representation, in_frame_units=False,
             (not an instance), or the string name of the representation
             class.
 
-        in_frame_units : bool
-            Force the representation units to match the specified units
-            particular to this frame
-
-        new_differential : subclass of `~astropy.coordinates.BaseDifferential`, str, optional
+        new_differentials : subclass of `~astropy.coordinates.BaseDifferential`, str, optional
```

Comment:
Why the plural? In principle, I guess we may later want to support more, so in that sense OK. But then, shouldn't we at least allow a `dict` here? (for now only with `'s'` as a possible key).
...  Ah, I see you do accept a dict below. So, just update the documentation here.